example/odoo/Dockerfile-onbuild

```diff
@@ -0,0 +1,15 @@
+FROM camptocamp/odoo-project:10.0-latest-onbuild
```

Comment:
worth to mention when this option shall be used?

Reply:
hmm maybe better in common/Dockerfile-onbuild

Reply:
Adding a section in the readme that explain the difference between the base version and the onbuild version? I can also explain the difference between the full version and normal version?

Reply:
the more you explain the better! thanks :heart: